continuous_integration/install.sh

```diff
@@ -75,4 +75,8 @@ if [ -z "$NUMPY_VERSION" ]; then
     mv setup.cfg{.new,}
 fi
 
+if [ ! -z "$DILL_VERSION" ]; then
```

Comment:
dill is available through conda, it would be good to install it like that. IIRC you just have to add dill to TO_INSTALL_MAYBE variable in continuous_integration/install.sh.